streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1836,6 +1841,33 @@ public void maybeThrowTaskExceptionsFromProcessingThreads() {
         }
     }
 
+    // track the size of the transaction buffer on each iteration to predict when it will be exceeded in advance
+    private long lastUncommittedBytes = 0L;
```

Comment:
Please move this field to the other fields at the top of the class.

Reply:
Done